jake/audit/audit.py

```diff
@@ -16,30 +16,59 @@
 import logging
 
 from typing import List
+from textwrap import wrap
 from colorama import Fore
+from terminaltables import DoubleTable
 
 from ..types.coordinateresults import CoordinateResults
 from ..types.vulnerabilities import Vulnerabilities
 
-class Audit():
+class Audit:
   """ Audit does the business, it prints results from OSS Index to the standard out """
+
   def __init__(self, quiet=False):
-    self._log = logging.getLogger('jake')
+    self._log = logging.getLogger("jake")
     self._quiet = quiet
 
   def audit_results(self, results: List[CoordinateResults]):
     """
     audit_results is the ingest point for the results from OSS Index,
     and handles control flow
     """
-    self._log.debug("Results recieved, %s total results", len(results))
+    self._log.debug("Results received, %s total results", len(results))
 
     total_vulns = 0
     pkg_num = 0
+    good = [x for x in results if len(x.get_vulnerabilities()) == 0]
+    bad = [x for x in results if len(x.get_vulnerabilities()) > 0]
+
+    if len(good) != 0:
+      print()
+      print("Non-Vulnerable Dependencies")
+      print()
+      for coordinate in good:
+        pkg_num += 1
+        total_vulns += self.print_result(coordinate, pkg_num, len(results))
+
+    if len(bad) != 0:
+      print()
+      print("Vulnerable Dependencies")
+      print()
+
+      for coordinate in bad:
+        pkg_num += 1
+        total_vulns += self.print_result(coordinate, pkg_num, len(results))
+
+    table_data = [
+        ["Audited Dependencies", len(results)],
+        ["Vulnerablities Found", total_vulns],
+    ]
 
-    for coordinate in results:
-      pkg_num += 1
-      total_vulns += self.print_result(coordinate, pkg_num, len(results))
+    table_instance = DoubleTable(table_data, "Summary")
+
+    print()
+
+    print(table_instance.table)
 
     return total_vulns
```

Comment:
Is it silly that the little unit test: `test/test_audit.py->test_call_audit_results_prints_output()`  that verifies the 3 return count makes me so happy? ;)